.github/workflows/build_and_test.yml

```
@@ -0,0 +1,907 @@
+name: "Build & Test"
+
+on:
+  push:
+    branches:
+      - master
+      - branch-DDS-3.15
+      - branch-DDS-3.14
+      - branch-DDS-3.13
+      - gh_workflow
```

```suggestion
      - gh_workflow
    paths:
      - '**'
      - '!docs/**'
      - '!README.md'
      - '!INSTALL.md'
      - '!NEWS.md'
```

This makes it not run the workflow if those are the only files that changed. We can add more if needed.